manifests/external-dns-values.yml.tpl

```diff
@@ -1,4 +1,14 @@
-fullnameOverride: external-dns
+podAnnotations:
+  azure.workload.identity/inject-proxy-sidecar: "true"
```

Comment:
A lot of these values are old, they were for the bitnami chart. I've updated it to the official external-dns helm chart (https://artifacthub.io/packages/helm/external-dns/external-dns) and WI.

Reply:
There was a merge issue which duplicated some values that I have removed.  Otherwise, the azure: block is required for private DNS and is ignored for public.

Reply:
Redundant config removed